Time and Time Again/src/TimeAndTimeAgain.java

```diff
@@ -0,0 +1,40 @@
+import java.io.File;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+import java.io.File;
+import java.io.IOException;
+
+public class TimeAndTimeAgain
+	{
+		public static void main(String[] args) throws IOException
+			{
+			Scanner input = new Scanner(new File("TimeAndTimeAgainTextfile.txt"));
+			int numberOfCases = input.nextInt();
+			for (int i = 0; i < numberOfCases; i++)
+				{
+				int hour = 0;
+				int minute = 0; 
+				int second = 0;
+				String data = input.nextLine();
+				
+				Pattern secondRegex = Pattern.compile(".*(//d+)s.*");
```

Comment:
So here you'll need to change the `//d+` to `\\d+` for the regex string pattern.  I think `//d` looks for a "//d+" instead of one or more digits, so that could be why when entering a number like 8s it doesn't read it in. Same for minute and  hour below.